src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ChangeRequestTab.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Grid, Typography, useTheme } from '@mui/material';
```

Comment:
A lot of this code is a copy from the ChangeRequestOveriew. We want to abstract this component and use props to change how the component is rendered for different uses.